src/transforms/samplesTransforms.ts

```diff
@@ -139,13 +139,16 @@ export async function getAllExamples(codeModel: TestCodeModel, clientDetails: Cl
             );
             const parameterTypeName = parameterTypeDetails.typeName;
             let paramAssignment = "";
+            let isAnyTypeBody = false;
             if (methodParameter.parameter.protocol?.http?.["in"] === "body") {
               sample.hasBody = true;
-              sample.bodySchemaName = parameterTypeName;
-              sample.importedTypes?.push(parameterTypeName);
+              sample.bodySchemaName = parameterTypeName;   
```

Comment:
What is the goal of this change?

Reply:
This is trying to fix some bug in the sample generation when the body schema type is any type or any object type, it will still  trying to import this Record<string, unknown> from the client. which doesn't exist at all. 
I guess I will probably need to file another PR for this. Will undo the change in this PR.